gofer.spec

```diff
@@ -73,6 +74,7 @@ cp usr/lib/systemd/system/* %{buildroot}/%{_unitdir}
 %else
 cp etc/init.d/%{name}d %{buildroot}/%{_sysconfdir}/init.d
 %endif
+cp etc/sysconfig/%{name}d %{buildroot}/%{_sysconfdir}/sysconfig
```

Comment:
Suggest this be moved to just after line 68 so it's grouped with other configuration.